src/MensaWidget/MWCanteen.class.st

```diff
@@ -33,20 +33,76 @@ Class {
 	#'squeak_changestamp' : 'Louis Christ 5/31/2024 14:42'
 }
 
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'Oscar 6/5/2024 00:26'
+}
+MWCanteen class >> canteenMorphBorder [
+
+	^ 5
+]
+
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'Oscar 6/5/2024 12:44'
+}
+MWCanteen class >> canteenMorphColor [ 
+
+	^ Color fern lighter
+]
+
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'tmc 6/6/2024 21:26'
+}
+MWCanteen class >> canteenMorphWidth [
+	
+	^ 410
+]
+
+{
+	#category : #constants,
+	#'squeak_changestamp' : 'tmc 6/6/2024 21:57'
+}
+MWCanteen class >> cellGap [
+
+	^ 5
+]
+
 {
 	#category : #'constructor ',
-	#'squeak_changestamp' : 'Louis Christ 5/29/2024 18:34'
+	#'squeak_changestamp' : 'tmc 6/6/2024 21:28'
 }
 MWCanteen class >> newWithId: anId Name: aName Date: aDate Meals: aMealList [
 
-	| newInstance | 
-	
-	newInstance := self new.
-	newInstance id: anId;
+	^ self new
+		id: anId;
 		name: aName;
 		date: aDate;
-		meals: aMealList.
-	^ newInstance.
+		meals: aMealList;
+		yourself
+	
+]
+
+{
+	#category : #display,
+	#'squeak_changestamp' : 'Louis Christ 6/6/2024 19:38'
+}
+MWCanteen >> addMealsTo: aCanteenMorph [
+
+	self meals do: [ :each | aCanteenMorph addMorph: each getMealMorph]
+]
+
```

Comment:
linebreak here after `... :each |` ?
Because we do so in  fillContentMorph

Reply:
also a space is needed before "]" if we continue using this policy